sechub-sarif/src/test/resources/examples/microsoft/sarif-tutorials/samples/3-Beyond-basics/bad-eval-with-code-flow.py

```diff
@@ -0,0 +1,8 @@
+# 3-Beyond-basics/bad-eval-with-code-flow.py
+
+print("Hello, world!")
+expr = input("Expression> ")
+use_input(expr)
+
+def use_input(raw_input):
+    print(eval(raw_input))
```

Comment:
This is a real vulnerability, do we want to have this in the SecHub code base?

Reply:
Hmm... its from the original microsoft sarif examples. We do not really need the origin sources. I will remove all origin source snippets - the sarif examples are enough. thanx.

Reply:
done: dropped both python example snippets